src/js/routes/services.js

```diff
@@ -24,7 +25,15 @@ let serviceRoutes = {
         {
           type: Route,
           name: 'services-detail',
-          path: ':id/?'
+          path: ':id/?',
+          children: [
+            {
+              type: Route,
+              name: 'services-task-panel',
```

Comment:
We should rename this route to a more appropriate `services-task-detail` now that the `panel` is gone.